hips/draw/tests/test_simple.py

```diff
@@ -4,45 +4,53 @@
 import healpy as hp
 from astropy.coordinates import SkyCoord
 from numpy.testing import assert_allclose
-from astropy.utils.data import get_pkg_data_filename
 from astropy.tests.helper import remote_data
 from ..simple import make_sky_image, draw_sky_image, compute_matching_hips_order, _get_hips_order_for_resolution
 from ...tiles import HipsSurveyProperties, HipsTileMeta, HipsTile
 from ...utils import WCSGeometry
 from ...utils.testing import get_hips_extra_file, make_test_wcs_geometry, requires_hips_extra
 
-def get_test_tiles():
-    filename = get_pkg_data_filename('../../tiles/tests/data/properties.txt')
+
+def get_test_tiles(file_format, survey, order, ipix_list):
+    filename = get_hips_extra_file('datasets/samples/' + survey + '/properties')
     hips_survey = HipsSurveyProperties.read(filename)
 
-    tile1 = HipsTile.read(
-        meta=HipsTileMeta(order=3, ipix=450, file_format='fits', frame=hips_survey.astropy_frame),
-        full_path=get_hips_extra_file('datasets/samples/DSS2Red/Norder3/Dir0/Npix450.fits'),
-    )
+    tiles = []
+    for ipix in ipix_list:
+        tiles.append(HipsTile.read(
+            meta=HipsTileMeta(order=order, ipix=ipix, file_format=file_format, frame=hips_survey.astropy_frame),
+            full_path=get_hips_extra_file('datasets/samples/' + survey + '/Norder' + str(order) + '/Dir0/Npix' + str(ipix) + '.' + file_format),
```

Comment:
Shouldn't building this string be a property or method somewhere, e.g. on `HipsTileMeta`?
```
survey + '/Norder' + str(order) + '/Dir0/Npix' + str(ipix) + '.' + file_format
```
Otherwise we duplicate that code 100 times in our repo.